prymer/primer3/primer3_parameters.py

```diff
@@ -92,6 +138,11 @@ class Primer3Parameters:
     primer_max_dinuc_bases: int = 6
     avoid_masked_bases: bool = True
     number_primers_return: int = 5
+    primer_max_self_any: float = 8.00
+    primer_max_self_any_thermo: float = field(init=False)
+    primer_max_self_end: float = 3.0
+    primer_max_self_end_thermo: float = field(init=False)
+    primer_max_hairpin_thermo: float = field(init=False)
```

Comment:
This feels like a rather unrelated addition - is there a reason this is part of this PR?

Reply:
these have been removed in the latest commit per our Slack discussion